app/frontend/src/components/admin/inspection/LocationEdit.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <v-dialog v-model="dialog" max-width="800px">
+    <template v-slot:activator="{ on }">
+      <v-btn v-if="true" v-on="on" color="primary" class="my-3" small>Edit Submission</v-btn>
+    </template>
+    <v-card>
+      <v-form ref="form" v-model="updateValid">
+        <v-card-title>
+          <span class="headline">
+            Editing submission from:
+            <strong>{{businessName}}</strong>
+          </span>
+        </v-card-title>
+        <v-card-text>
+          <v-container class="pa-0">
+
+            <!-- use Location form component -->
+            <Location :reviewMode="false"/>
+
+          </v-container>
+        </v-card-text>
+        <v-card-actions class="pa-3 pb-10">
+          <v-btn color="primary" @click="submit">Save</v-btn>
+          <v-btn text @click="cancelUpdate">Cancel</v-btn>
+        </v-card-actions>
+      </v-form>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script>
+import { mapGetters, mapActions } from 'vuex';
+//import form from '@/store/modules/form.js';
+
+import Location from '@/components/form/Location.vue';
+import ipcService from '../../../services/ipcService';
+
+export default {
+  name: 'LocationEdit',
+  components: {
+    Location,
+  },
+  props: {
+    ipcPlanId: {
+      required: true,
+      type: String
+    },
+  },
+  data() {
+    return {
+      dialog: false,
+      updateValid: false,
+      reviewMode: false,
+      updateComplete: false,
+      updateError: '',
+      startDateMenu: false,
+      endDateMenu: false
+    };
+  },
+  computed: {
+    ...mapGetters('form', ['business', 'contacts', 'covidContact', 'ipcPlan', 'location']),
```

Comment:
If you mount the form store in the namespace "edit", your mapgetter needs to pull from the "edit" namespace, not "form". This referencing form instead is why it doesn't look like lazy loading is working.